addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -1027,6 +1027,12 @@ const ColorpickerUserValueWidget = SelectUserValueWidget.extend({
     isContainer: function () {
         return false;
     },
+    /**
+     * @override
+     */
+    isActive: function () {
+        return this._value !== 'rgba(0, 0, 0, 0)';
```

Comment:
Use the function to compare colors which is defined somewhere. web_editor.utils / colorpicker ? Even if _value always is `rgba(0, 0, 0, 0)` for transparent colors, it would be more robust.